earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -64,6 +66,32 @@ main_postinstall()
     final_fusion_service_configuration
 }
 
+
+
+create_users_and_groups()
+{
+    local GROUP_EXISTS=$(getent group "$GEGROUP")
+
+    # Add group if it does not exist.  Note that it is also created by the common installer, 
+    # and common will remove it if necessary. 
+    if [ -z "$GROUP_EXISTS" ]; then
+        groupadd -r "$GEGROUP" &> /dev/null
+        echo "Fusion install added group $GEGROUP"
+    fi
+
+    # Add user if it does not exist:
+    if [ -z "$(getent passwd "$GEFUSIONUSER")" ]; then
+		mkdir -p "$BASEINSTALLDIR_OPT/.users/$GEFUSIONUSER"
```

Comment:
Minor: there's some funny indentation in this if/else statement.  It would be nice to clean it up.